doc/reference/network_bridge.md

```diff
@@ -6,39 +6,50 @@ discourse: 7322
 # Bridge network
 
 As one of the possible network configuration types under LXD, LXD supports creating and managing network bridges.
+<!-- Include start bridge intro -->
+A network bridge creates a virtual L2 ethernet switch that instance NICs can connect to, making it possible for them to communicate with each other and the host.
 LXD bridges can leverage underlying native Linux bridges and Open vSwitch.
```

Comment:
This line is from the original documentation (which is probably from before OVN support) - is it still correct?

Reply:
This is still correct AFAIK, the Open vSwitch reference is still valid (it can create or use an Open vSwitch bridge) and is distinct from OVN networks.